lib/render.js

```diff
@@ -27,7 +28,8 @@ function renderChart() {
         // reset and remove it
         __dw.vis.reset();
     }
-    const $chart = document.querySelector('.dw-chart-body');
+
+    $chart = document.querySelector('.dw-chart-body');
```

Comment:
can we keep this a `const` for now?

Reply:
I moved `$chart` one step up in the block scope  because it is also needed for initializing the `resize` events, which is no longer done in every render cycle. But if you think that's an issue, we could leave it a local `const` in the render function and get another reference for initializing the resize observer.